.github/workflows/_binary-upload.yml

```diff
@@ -59,18 +59,17 @@ on:
       github-token:
         required: true
         description: Github Token
-      aws-pytorch-uploader-access-key-id:
-        required: true
-        description: AWS access key id
-      aws-pytorch-uploader-secret-access-key:
-        required: true
-        description: AWS secret access key
       conda-pytorchbot-token:
         required: true
         description: Conda PyTorchBot token
       conda-pytorchbot-token-test:
         required: true
         description: Conda PyTorchBot token
+
+permissions:
+  id-token: write
+  contents: read
```

Comment:
For this to work, caller should pass this token, shouldn't it?

Reply:
Yes adreessed